index.md

```diff
@@ -1,38 +1,53 @@
 ---
 layout: homepage
 title: sgID
-description: Designed for privacy
+description: An Isomer site of the Singapore Government
 image: /images/isomer-logo.svg
 permalink: /
-notification: null
+notification: ""
 sections:
   - hero:
       title: sgID
-      subtitle: sgID is an authentication and data sharing service <br>focused on
-        preserving privacy.
       background: /images/background.png
-      button: Contact Us
-      url: https://go.gov.sg/sgid-contact
+      dropdown:
+        title: Shortcuts
+        options:
+          - title: Documentation
+            url: https://docs.id.gov.sg
+          - title: Developer Portal
+            url: https://developer.id.gov.sg
+          - title: Contact Us
+            url: https://go.gov.sg/sgid-contact
+      subtitle: A Singapore government identity provider with a focus on citizen privacy
   - infopic:
-      title: Familiar authentication flow
-      description: sgID is fully integrated into the Singpass App. Simply scan the QR
-        code with the Singpass app to log in with sgID.
+      title: Identity with a focus on privacy
+      description: sgID is a Singapore government service that allows Singapore
+        residents to perform end-to-end encrypted logins to applications with
+        the Singpass mobile app
```

Comment:
nit: i think we should have a fullstop here

Reply:
done!